helm/ph-ee-engine/operations-web/values.yaml

```diff
@@ -44,6 +44,9 @@ backend:
   PH_OPS_BACKEND_SERVER_URL: https://ops-bk.sandbox.mifos.io/api/v1
   PH_VOU_BACKEND_SERVER_URL: https://ops-bk.sandbox.mifos.io/api/v1
   PH_ACT_BACKEND_SERVER_URL: https://ops-bk.sandbox.mifos.io/api/v1
+  PH_VOU_CALLBACK_URL: https://vouchers.sandbox.fynarfin.io
```

Comment:
keep it as Mifos here and use fynarfin.io in fynarfin sit chart

Reply:
Updated